packages/components/README.md

```diff
@@ -41,6 +41,4 @@ in the root of the repository to get the required `devDependencies`.
 
 ### Using [Storybook](https://storybook.js.org/)
 
-To see stories within this package, run `yarn workspace @automattic/components run start-storybook`.
-
-To see all stories within this repository, run `yarn storybook:start` at the root of the repository.
```

Comment:
This is untrue, as the root Storybook only includes a subset:

https://github.com/Automattic/wp-calypso/blob/8be8e8b306c79d5a9969bb27d9a141a473457e50/.storybook/main.js#L6-L11